e2e/testdata/fn-eval/out-of-place-dir-exists-error-sink/.expected/config.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+exitCode: 1
+stdErr: |
+  directory "out" already exists, the resources will not be written to "out" even if the function run is successful, please delete the directory and retry
```

Comment:
Do we need the part `the resources will not be written to "out" even if the function run is successful` ?

`directory "out" already exists, please delete the directory and retry`
?

Reply:
Updated.